cmd/registry/cmd/apply/testdata/lifecycle.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apigeeregistry/v1
+kind: Lifecycle
+metadata:
+  name: lifecycle
+data:
+  displayName: Lifecycle
+  description: A series of stages that an API typically moves through in its lifetime
+  stages:
+  - id: concept
+    displayName: Concept
```

Comment:
Is there a reason why "Concept" stage doesn't have display order defined?

Reply:
Its display order was zero and the exporter wasn't exporting zero values because "omitempty" was declared on the YAML configuration. I've removed "omitempty" from the lifecycle and taxonomy "displayOrder" fields so they're now included.